sound/soc/intel/common/soc-acpi-intel-ptl-match.c

```diff
@@ -264,24 +264,6 @@ static const struct snd_soc_acpi_adr_device cs35l56_3_3amp_adr[] = {
 	}
 };
 
-static const struct snd_soc_acpi_adr_device cs42l43_3_adr[] = {
```

Comment:
Reordering `snd_soc_acpi_intel_ptl_sdw_machines[]` is enough. Let's leave other stuff as it is and help people focus on the required change.